gns/train.py

```diff
@@ -21,6 +21,17 @@
 from gns import data_loader
 from gns import distribute
 
+import random
+seed=42
+torch.manual_seed(seed)
+torch.cuda.manual_seed(seed)
+torch.backends.cudnn.deterministic = True
+random.seed(seed)
+np.random.seed(seed)
+torch.use_deterministic_algorithms(True)
+# do this on a bash shell
+# export CUBLAS_WORKSPACE_CONFIG=:4096:8
+
```

Comment:
```suggestion

```

Reply:
I suggest commenting these out, just in case someone wants reproducible outputs.